### DISCUSSION TITLE
[NEOVIM] Copilot setup in neovim broken

### DISCUSSION THREAD

**Discussion by nzambello** [community/community#17613] [Copilot Conversations] [answered]
Hi, I've installed the plugin but I get this error when I run `:Copilot setup`:

```
Error detected while processing function copilot#Command[3]..copilot#Call[2]..<SNR>176_AgentCall[1]..<SNR>176_AgentRequest[3]..<SNR>176_Send:
line    3:
E716: Key not present in Dictionary: "job"
E116: Invalid arguments for function ch_sendraw
```

My setup:
- macOS 12.4 on Apple Silicon M1
- Node 16.13.1
- Neovim 0.7.0
- github/copilot.vim release tag (1.3.3)

What I did:
- Installed `copilot.vim` with `vim-plug` 
  ```Plug 'github/copilot.vim'```
- Run `:PlugInstall` with no issues nor errors
- Got that error while running `:Copilot setup` or `:Copilot! setup`

@tpope do you have any suggestion, please? Thank you

**Comment by tpope** [comment, marked as answer]
Can you try the just released update?

**Reply by nzambello** [reply]
All right, now I've been able to set it up. It's now running and works great!
Thank you!!
